docs/manual/java/guide/cluster/PersistentEntityCassandra.md

```diff
@@ -22,4 +22,54 @@ In sbt:
 
 ## Configuration
 
-Lagom's Cassandra support is provided by the [`akka-persistence-cassandra`](https://github.com/akka/akka-persistence-cassandra) plugin. A full configuration reference can be in the plugins [`reference.conf`](https://github.com/akka/akka-persistence-cassandra/blob/v0.20/src/main/resources/reference.conf).
+Lagom uses several internal Cassandra tables to persist entity data. You need to configure the keyspaces that are used for these tables. By default, Lagom automatically creates these keyspaces and tables at startup if they're missing.
```

Comment:
OK done. I made some minor tweaks to the wording or spelling of your suggestion. Let me know if I missed something or didn't correctly capture your intent.